cypress/objects/timeLoggingPage.js

```diff
@@ -0,0 +1,38 @@
+class TimeLoggingPage {
+
+    
+    getPageTitle(){
```

Comment:
I suppose you could remove the "get" at the beginning to make this function's name consistent with all the others :)